src/BibleWell.App/ViewModels/OpenFileViewModel.cs

```diff
@@ -0,0 +1,70 @@
+using Avalonia.Media;
```

Comment:
Unused directive is unnecessary error. Removing allows Android to build but then clicking the Maui implementation button crashes the app.

Reply:
Okay, for some reason I had to re-install maui essentials, and apparently it made a change to the Props file thing, so I've pushed that up